src/cmd/create.go

```diff
@@ -463,7 +463,11 @@ func getEnterCommand(container, release string) string {
 	case containerNamePrefixDefaultWithRelease:
 		enterCommand = fmt.Sprintf("%s enter --release %s", executableBase, release)
 	default:
-		enterCommand = fmt.Sprintf("%s enter --container %s", executableBase, container)
+		enterCommand = fmt.Sprintf("%s enter %s | %s enter --container %s",
```

Comment:
I think it's better to just mention the new syntax, and completely leave out the old one. :)

We still support the old syntax in the code for the sake of compatibility with old documentation and old habits. We should switch our user interface, documentation and such to advertise the new syntax, so that finally after some time down the road we can remove the old syntax.